src/panel.ts

```diff
@@ -114,8 +116,40 @@ export class HistoryDebuggerPanel {
       // TODO: error handling
       switch (e.type) {
         case "startFromId": {
-          console.log()
+          // TODO: Implement history pagination
+          const conn = await Connection.connect(/* { address: 'temporal.prod.company.com' } */)
+          const { history } = await conn.workflowService.getWorkflowExecutionHistory({
+            namespace: "default",
+            execution: {
+              workflowId: e.workflowId,
+              runId: e.runId,
+            },
+          })
+          if (!history) {
+            await vscode.window.showErrorMessage("Empty history")
+            break
+          }
+          const bytes = new Uint8Array(temporal.api.history.v1.History.encodeDelimited(history).finish())
+          this.currentHistoryBuffer = Buffer.from(bytes)
+          await webview.postMessage({ type: "historyProcessed", history: bytes })
+          // TODO: Send history file to local server
           await vscode.window.showInformationMessage("Starting debug session")
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          const _config = { env: { TEMPORAL_DEBUGGER_PLUGIN_URL: this.httpServerUrl } }
```

Comment:
You sould put the env directly in the config otherwise you'l get a nested `env.env`